package.json

```diff
@@ -204,6 +204,7 @@
     "rollup-plugin-visualizer": "^5.6.0",
     "serve-handler": "^6.1.5",
     "storybook": "7.0.0-rc.5",
+    "storybook-addon-apollo-client": "^4.1.4",
```

Comment:
Wow I'm surprised they updated this package, in my Storybook upgrade PR I noticed it seemed unmaintained https://github.com/evergreen-ci/spruce/pull/1739. Do you prefer it to adding the `MockedProvider` directly to stories that require it?

If we go with the addon, we should remove the `MockedProvider` from stories that have it included.